sml/metrics/classification/classification_test.py

```diff
@@ -33,16 +33,100 @@
 from sml.metrics.classification.classification import (
     accuracy_score,
     average_precision_score,
+    balanced_accuracy_score,
     bin_counts,
     equal_obs,
     f1_score,
     precision_score,
     recall_score,
     roc_auc_score,
+    top_k_accuracy_score,
 )
 
 
 class UnitTests(unittest.TestCase):
+
+    def test_balanced_accuracy(self):
+        sim = spsim.Simulator.simple(
+            3, spu_pb2.ProtocolKind.ABY3, spu_pb2.FieldType.FM128
+        )
+
+        def proc(y_true: jnp.ndarray, y_pred: jnp.ndarray, labels: jnp.ndarray):
+            balanced_score = balanced_accuracy_score(y_true, y_pred, labels)
+            return balanced_score
+
+        def sklearn_proc(y_true, y_pred):
+            balanced_score = metrics.balanced_accuracy_score(y_true, y_pred)
+            return balanced_score
+
+        def check(spu_result, sk_result):
+            np.testing.assert_allclose(spu_result, sk_result, rtol=1, atol=1e-5)
+
+        # Test binary
+        y_true = jnp.array([0, 1, 1, 0, 1, 1])
+        y_pred = jnp.array([0, 0, 1, 0, 1, 1])
+        labels = jnp.array([0, 1])
+        spu_result = spsim.sim_jax(sim, proc)(y_true, y_pred, labels)
```

Comment:
Test the param `sample_weight` and `adjusted`;
Test larger datasets, please (maybe ~1000 samples are enough).